pkg/api/cluster_handler.go

```diff
@@ -71,16 +71,17 @@ func (e *etcdHandler) GetObjectStoreConnectionInfo(name string) (*model.ObjectSt
 		return nil, true, err
 	}
 
-	host, ipEndpoint, found, err := rgw.GetRGWEndpoints(e.context.EtcdClient, clusterInventory)
+	host, ipaddress, found, err := rgw.GetRGWEndpoints(e.context.EtcdClient, clusterInventory)
```

Comment:
is this function's name misleading now?

Reply:
yeah, but this will go away with standalone so i won't worry about it